src/ui/server/constants/api.ts

```diff
@@ -31,7 +31,8 @@ export const API = {
 
 export const EXTERNAL_API_ENDPOINTS = {
   MULESOFT_MDM_EA: {
-    CURRENCY: '/currency',
-    INDUSTRY_SECTORS: '/map-industry-sector?size=1000',
+    CURRENCY: '/currencies',
+    INDUSTRY_SECTORS: '/sector-industries',
+    MARKETS: '/markets?active=Y',
```

Comment:
Active flag might not be required here, in the eligibility areas we want to display inactive/unsupported countries so we can render an appropriate message if a user selects an inactive/unsupportred country

Reply:
Have removed query parameter `active` with value `Y`